packages/compass-sidebar/src/components/multiple-connections/connections-navigation.tsx

```diff
@@ -497,6 +503,12 @@ const ConnectionsNavigation: React.FC<ConnectionsNavigationProps> = ({
   );
 };
 
+const onRefreshDatabases = (): SidebarThunkAction<void> => {
+  return (_dispatch, getState, { globalAppRegistry }) => {
+    globalAppRegistry.emit('refresh-databases');
```

Comment:
As far as I can tell this refresh-databases and the one on the Databases tab where I copied it from will will refresh for all connections. Assuming I'm reading it correctly.

Reply:
yea you will need to add connectionId to this most likely as second parameter.

Reply:
and I think in the databases-collection, that's also a problem, we will also have to update the corresponding check in compass-app-stores which actually refreshes the database. Sorry I must have either missed this when I moved compass-app-stores events or accidentally skipped it.

Reply:
Should be good now?